tests/integration/framework/process/daprd/daprd.go

```diff
@@ -80,7 +80,7 @@ func New(t *testing.T, fopts ...Option) *Daprd {
 		publicPort:       fp.Port(t),
 		metricsPort:      fp.Port(t),
 		profilePort:      fp.Port(t),
-		logLevel:         "info",
+		logLevel:         "debug",
```

Comment:
Will this get put back to `info` after further flaky test squashing?
```suggestion
		logLevel:         "info",
```

Reply:
I think it's correct for us to keep it debug actually. When a test fails a dev will wish that debug was set, so it makes sense to have that default.

The only thing we lose having this as debug is the extra bit of RAM holding the extra debug log lines that will get de-referenced and quickly garbage collected after that test case is successful.

Reply:
makes sense to me, and will be helpful for us later 👍🏻